functions/Export-SqlUser.ps1

```diff
@@ -96,7 +96,8 @@ https://dbatools.io/Export-SqlUser
 		[System.Management.Automation.PSCredential]$SqlCredential,
 		[Alias("NoOverwrite")]
 		[switch]$NoClobber,
-		[switch]$Append
+		[switch]$Append,
+		[switch]$IncludeSystemDatabases = $false
```

Comment:
@ClaudioESSilva, @potatoqualitee: Acknowledged and understood. I'm inclined to change the default to `$true`.